### PR TITLE
Fix, add and remove some (wrong or missing) info

### PR DESCRIPTION
**Pull request by stonar96**
(See commit messages.)

### CODE REVIEW

**Comment by stonar96** [comment]
I have improved grammar a bit and documented every missing flag that exists.

File: source/regions/common-scenarios.rst [hunk 1]
```diff
@@ -70,25 +70,25 @@ How do I unprotect all enchantment tables?
...
 
 How do I allow players to read lectern books, but not take them?
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 First, make sure you are on WorldGuard 7.0.1+. Earlier versions of WorldGuard released before 1.14 (and thus before lecterns) existed.
 
-* Set the ``interact`` flag to ``allow``: ``/rg flag <region> interact allow``
-* Set the ``chest-access`` flag to ``deny``, at least for non-members: ``/rg flag <region> chest-access -g nonmembers deny``
+* Set the ``use`` flag to ``allow``: ``/rg flag <region> use allow``
+* Set the ``chest-access`` flag to ``deny``, at least for non-members (note that this is the default): ``/rg flag <region> chest-access -g nonmembers deny``
```
Comment (wizjany) [inline source/regions/common-scenarios.rst]:
```suggestion

```
This was written before I separated the use and chest-access flags, it's no longer necessary at all.

File: source/regions/global-region.rst [hunk 2]
```diff
@@ -42,16 +42,19 @@ Adding Members
 
 Due to legacy reasons, adding owners or members to the global region implicitly sets ``passthrough`` to deny. That means that you do not have to actually change the flag, although it is recommended anyway.
 
+Non Player Associables
+~~~~~~~~~~~~~~~~~~~~~~
+
+Non player associables, such as pistons for example, are usually members of either all regions in which they are in or only of the regions with the highest priorities in which they are in, depending on the ``use-max-priority-association`` :doc:`../config` setting. However, non player associables are no longer members of the global region, if there is at least one other region that they are in. This means that pistons for example cannot push blocks from inside a region into a protected global region. Thus the global region always behaves as if ``use-max-priority-association`` is set to ``true``.
```
Comment (wizjany) [inline source/regions/global-region.rst]:
```suggestion
Non-player associables, such as pistons, are usually members of either all regions in which they are in or only of the regions with the highest priorities in which they are in, depending on the ``use-max-priority-association`` :doc:`../config` setting. However, non-player associables are no longer members of the global region if they are in at least one other region. This means that pistons, for example, cannot push blocks from inside a region into a protected global region. Thus the global region always behaves as if ``use-max-priority-association`` is set to ``true``.
```

**Comment by stonar96** [comment]
Done, correct the same mistakes in other places too.

**Pull request merged**